### ISSUE TITLE
Support Disqus commenting by enabling configuration of EXAMPLE Disqus URL

### ISSUE THREAD

**Issue by jocelynpender** [peterdesmet/petridish#80]
I've created a Disqus site for my site. However, it's not functioning and I'm pretty confident that it's because `EXAMPLE` needs to be replaced by my site name on this [line](https://github.com/peterdesmet/petridish/blob/afabe47a1c25d2bef1c270791f4ba9d9fcdf3365/_layouts/default.html#L33) of the Petridish default layout.
I can certainly create my own version of the `default.html` layout that is unique to my site and separate from the theme, but I thought I would flag this for you so that it might be a configuration variable for folks in the future.

**Comment by peterdesmet** [comment]
Thanks for reporting, that is indeed a bug.

**Comment by peterdesmet** [comment]
I'm thinking of including the Disqus site config as one of the socials in `_config.yml`. Does that make sense?

```yml
socials:
  twitter: peterdesmet
  github: github_username
  disqus: disqus_sitename # https://<disqus_sitename>.disqus.com/
```

It would not be listed as one of the footer icons.

**Comment by jocelynpender** [comment]
Yes, this makes sense! I was thinking of the same idea for a solution 😄

**Comment by peterdesmet** [comment]
This is now fixed and retroactively included in 3.0. So you can still reference:

```
remote_theme: peterdesmet/petridish@3.0
```

The feature is documented at http://peterdesmet.com/petridish/docs/configuration/#blog-posts
